contracts/pcv/utils/StaticPCVDepositWrapper.sol

```diff
@@ -26,6 +28,8 @@ contract StaticPCVDepositWrapper is IPCVDepositBalances, CoreRef {
     constructor(address _core, uint256 _balance, uint256 _feiBalance) CoreRef(_core) {
         balance = _balance;
         feiReportBalance = _feiBalance;
+
+        _setContractAdminRole(keccak256("ORACLE_ADMIN_ROLE"));
```

Comment:
Interesting to re-use the oracle admin role, it's weird but it makes sense

Reply:
added a comment